### DISCUSSION TITLE
Takım görevleri 2 - Proje hazırlama

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/BilsemPython#82] [Göster ve Söyle]
Aşağıdaki üç tematik konuda her takımın en az 1 proje fikirlerini takımca karar alıp yapmasını istiyorum. Takım olarak birden fazla proje ile katılabilirsiniz. 

### STEAM

STEAM, gerçek dünya problemlerinin tanımlanması ve çözümünde fen bilimleri, teknoloji,
mühendislik, sanat ve matematik disiplinlerine özgü bilgi ve becerilerin disiplinler arası
yaklaşımla bir arada kullanılmasıdır. STEAM uygulamalarına; deprem merkez üssü ve fay
hatlarının gösterildiği topografik bir harita oluşturma, giyilebilir teknolojiler ile akıllı tekstil
ürünleri tasarlama, farklı teknolojiler aracılığıyla gerçek dünya problemlerinin çözümünde
kullanılabilecek fraktal yapılar oluşturma, beden kitle indeksi değeri dikkate alınarak bir
bireyin günlük besin ihtiyaçlarının yer alacağı beslenme çantası tasarımı örnek olarak
verilebilir.
STEAM konusunda; sanatın ve estetiğin göz ardı edilmediği, öğrencileri günlük hayat
problemlerinin çözümünde sıra dışı fikirler üretmeye teşvik ederek hayal gücü ve yaratıcılık
becerilerinin gelişmesine fırsat tanıyan STEAM disiplinlerinin entegrasyonuna yönelik
projeler hazırlanabilir. Ayrıca fen bilimleri, teknoloji, mühendislik, sanat ve matematik
disiplinlerinin entegrasyonuna dayanan günlük yaşam problemlerinin çözümünde bu
disiplinlerin etkili ve entegre bir şekilde kullanıldığı projeler sunulabilir. Öğrenciler teknoloji
(sensör, sesli ikaz cihazı, mikrodenetleyici kart), fen bilimleri (elektrik devreleri), mühendislik
(mühendislik tasarım döngüsü) ve matematik (hesaplama, geometri) disiplinlerinden
faydalanarak yangına ve gaz kaçağına duyarlı bir akıllı ev sistemi tasarlayabilir/geliştirebilir

### Sürdürülebilir Şehirler ve Toplumlar
21. yüzyılla birlikte şehirlerde büyük çaplı değişimler gerçekleşmektedir. Bu değişim
sürecinde sürdürülebilir ulaşım, akıllı şehirler, yeşil evler, ekolojik parklar gibi şehirlerin ve
toplumun yaşam kalitesi ve refahını artıracak çevre dostu unsurlara yer verilmektedir. Bir
taraftan da özellikle büyük şehirlerdeki artan nüfus ve enerji ihtiyacı, dağınık kentleşme,
doğal kaynakların tüketimi, kirlilik ve atık sorunları endişe vermektedir. Birleşmiş Milletler
2030 Sürdürülebilir Kalkınma Amaçları arasında da açıkça vurgulanan “sürdürülebilir şehirler
ve toplumlar” ile şehirlerin ve insan yerleşimlerinin kapsayıcı, güvenli, dayanıklı ve
sürdürülebilir kılınması hedeflenmektedir.
Sürdürülebilir şehirler ve toplumlar konusunda; güvenli ve erişilebilir konutlar, erişilebilir ve
sürdürülebilir ulaşım sistemleri, kapsayıcı ve sürdürülebilir kentleşme, kültürel ve doğal
dünya mirasının korunması, doğal afetlerin olumsuz etkilerinin azaltılması, şehirlerin çevresel
etkilerinin azaltılması, güvenli ve kapsayıcı yeşil alanlara ve kamusal alanlara erişimin
sağlanması ile ilgili projeler gerçekleştirilebilir. 

### Oyun ve Oyunlaştırma
Oyunlaştırma, herhangi bir sürecin rozet verme, ödüllendirme, geri bildirimler ya da
hikâyeleştirme gibi motive edici unsurlarla donatılması anlamına gelen yeni bir uygulama
alanıdır. Bir sınıftaki öğrencilerden olumlu davranışta bulunanlara artı, olumsuz davranışta
bulunanlara eksi puan vererek sene sonunda en fazla puan alan öğrenciye hediye verilmesi bir
oyunlaştırma örneği olarak sunulabilir. Nitekim sınıf ortamındaki öğrenciler için bir oyun
tasarlanmış ve davranışları sonucunda ödül verilmiştir. Oyunun uygulanacağı ortam (sınıf, ev
vb.) ve oyun dinamiklerinden hangilerinin kullanılacağı, sunulacak olan projeye göre
değişkenlik gösterebilmektedir. Bu kapsamda oyun dinamiklerini kullanarak bir oyun
tasarlamaya yönelik projelerin tümü bu başlık altında ele alınabilir.
Bu teknikler aynı zamanda oyun dinamiklerinin oyun dışı ortamlarda kullanılması anlamına
da gelmektedir. Bu bağlamda, spor aktiviteleri gibi zorlayıcı ve güdülenme gerektiren
etkinliklerin daha eğlenceli hale getirilmesi olanaklı olabilmektedir. Oyunlaştırma konusu,
insanlar için faydalı olabilecek spor ya da diyet yapma, öğrenme, sınavlara hazırlanma gibi
çeşitli süreçleri eğlenceli hale getirecek, oyunlaştırma dinamiklerinin kullanılacağı tasarımlar
yapmaya yönelik bir konudur.

Seçtiğiniz bir temada proje üreteceksiniz.

- Proje adı (ve takım adınız)
- Tema konusu
- Amaç
- Yöntem
- Beklenen sonuçlar

şeklinde beş bölümde detaylı açıklamalarınızı bu konu altına bekliyorum. Her geçerli proje fikri takımlarınıza 20 puan kazandırır.

**Comment by sonsuzus** [comment]
[4006-b_kilavuz-ocak2023.pdf](https://github.com/sonsuzus/BilsemPython/files/10685051/4006-b_kilavuz-ocak2023.pdf)

Proje geliştirirken bu kılavuz işinize yarayacaktır.

**Comment by adasue** [comment]
# Yangın Kaçış Kapısı (MONSTERs Takımı)

### Tema Konumuz:
Yangın ve yangından korunmanın yolları
### Amacımız:
Bu projede amacımız yangın sebepli ölümleri, yaranmaları azaltmak; yangın anında neler yapılması gerektiğiyle ilgili insanları bilinçlendirmek.
### Yöntemimiz:
Bu devrenin tasarım aşamasında şu adımları izledik: Tinkercad sitesinin devreler sekmesinde bulunan ArduinoUno seçeneğini kullanarak öncelikle yangın anında kişinin dikkatini çekmesi için kırmızı bir led ekledik. Anot ucundun 220 ohmluk dirence bağladık. Direnci de 9. analog pinine bağladık. Daha sonrasında devrenin yangın anını fark etmesi için bir gaz sensörü yerleştirdik. Onu da 10 kiloohmluk dirence bağladık. Yangın anında kişinin ledi görememesi ihtimaline karşın kişinin dikkatini çekmek adına bir bazr ekledik. Onu da 3. analog pinine bağladık. Yangın anında kişinin hızlı bir şekilde yangın henüz büyümeden önce bulunduğu alandan ayrılması için bir yangın kaçış kapısı ekledik. Onu da 5. analog pinine bağladık. Devrenin C++ kodları  şu şekildedir:

```c
#include <Servo.h>

int gaz_seviyesi = 0;

int gaz_seviyesi2 = 0;

Servo servo_5;

void setup()
{
  pinMode(A0, INPUT);
  servo_5.attach(5, 500, 2500);
  pinMode(3, OUTPUT);
}

void loop()
{
  gaz_seviyesi = analogRead(A0);
  if (gaz_seviyesi > 500) {
    servo_5.write(90);
    tone(3, 523, 500); // play tone 60 (C5 = 523 Hz)
    tone(3, 2960, 500); // play tone 90 (F#7 = 2960 Hz)
  } else {
    servo_5.write(0);
    noTone(3);
  }
  delay(10);
```

### Beklediğimiz Sonuçlar:
Bu projedeki ana fikrimizin günlük hayatta da yaygın ve kullanılır hale gelmesini sağlamak ve bu konuyla ilgili bir farkındalık yaratmak.

![Yangın Kaçış Kapısı Arduino2](https://user-images.githubusercontent.com/114185391/226405004-79828d53-961b-463d-aa17-780ecd9fd1fa.png)


![Yangın Kaçış Kapısı Arduino](https://user-images.githubusercontent.com/114185391/226404369-023a8f4c-7c58-4704-8907-329b6114a21a.png)
